templates/node-class.mustache

```diff
@@ -14,6 +14,15 @@ var {{&className}} = (function(){
         this.domain = domain;
     }
 
+    {{#isSecure}}
```

Comment:
@wcandillon Well, it can be as an option. We still need to ability to inject it because:
1. In the lifetime of the API class the token can be updated few times. For example: token's lifetime is a day and the browser tab is being open for a week -> it still should be possible to use the API.
2. One of the API's methods is GIVE_ME_THE_TOKEN so the token will be known after construction :)

What do you think ?